src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -10,6 +10,48 @@
     {{ parent() }}
 {% endblock %}
 
+{% block head_additions %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/monokai_sublime.min.css">
+    <script type="text/javascript">
+      WebFontConfig = {
+        google: { families: [ 'Source+Code+Pro::latin' ] }
+      };
+      (function() {
+        var wf = document.createElement('script');
+        wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
+        wf.type = 'text/javascript';
+        wf.async = 'true';
+        var s = document.getElementsByTagName('script')[0];
+        s.parentNode.insertBefore(wf, s);
+      })();
+      function highlight_code()
+      {
+        if (typeof (Worker) === undefined) return false;
+
+        var workerFunction = new Blob(['(' + highlight_code_worker_function.toString() + ')()'], {type: "text/javascript"});
+        var localWorkerURL = URL.createObjectURL(workerFunction);
+
+        $('div.readme pre, div.readme code').each(function() {
+          var code = $(this);
+          var worker = new Worker(localWorkerURL);
```

Comment:
Ok, seems the caching behaviour is my fault. `disable cache (while DevTools) open` was turned on.